src/components/molecules/Visualizer/Engine/Cesium/hooks.ts

```diff
@@ -60,6 +60,16 @@ export default ({
   // imagery layers
   const [imageryLayers, setImageryLayers] = useState<ImageryLayerData[]>();
 
+  // indicator
+  const selectionIndicator = useMemo(() => {
+    if (
+      property?.indicator?.indicator_type == undefined ||
+      property?.indicator?.indicator_type == "default"
+    )
+      return true;
+    return false;
+  }, [property?.indicator?.indicator_type]);
```

Comment:
This can be cleaned up like so:
```
  const selectionIndicator = useMemo(() => {
!!(property?.indicator?.indicator_type == undefined || property?.indicator?.indicator_type == "default");
  }, [property?.indicator?.indicator_type]);
```